llvm/test/CodeGen/NVPTX/fexp2.ll

```diff
@@ -0,0 +1,47 @@
+; RUN: llc < %s -march=nvptx64 -mcpu=sm_52 -mattr=+ptx86 | FileCheck --check-prefixes=CHECK %s
+; RUN: %if ptxas-12.6 %{ llc < %s -march=nvptx64 -mcpu=sm_52 -mattr=+ptx86 | %ptxas-verify -arch=sm_52 %}
```

Comment:
optional: Nothing wrong with using 12.6 version,
Can we use a lesser ptx version since the instruction is available from sm_52 onwards (to get wider coverage)?
(let's say 12.0 or something)

Reply:
Changed to `ptxas-12.0` and using PTX 1.4.